### DISCUSSION TITLE
Error when running Windows to-do app

### DISCUSSION THREAD

**Discussion by sam-wheat** [unoplatform/uno#10574] [Q&A]
I'm trying to run the [to-do app](https://github.com/unoplatform/uno.todo).

App works when I set startup to Skia.Gtk, Skia.WPF, or Wasm.  However when I set startup to Windows I get this error:

> Message=Unable to load DLL 'Microsoft.ui.xaml.dll' or one of its dependencies: The specified module could not be found. (0x8007007E)

A quick google led me add this to the .csproj:

    <WindowsAppSDKSelfContained>true</WindowsAppSDKSelfContained>

Which gives this error:

> Error	NETSDK1152	Found multiple publish output files with the same relative path: 

Another google led me add this (albeit a workaround):

    <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>

Which gives me this error:

>  Error	APPX1101	Payload contains two or more files with the same destination path 'Microsoft.Web.WebView2.Core.dll'. 

I suspect I've entered a rabbit hole that I should not be in in the first place so before I start aimlessly whacking at things I will seek some help here.

Windows 10 Pro  21H2 build 19044.2251
Visual Studio 17.4.1

**Comment by jeromelaban** [comment]
It's likely that the template is missing pubxml files. Can you [try copying this folder](https://github.com/unoplatform/uno/tree/master/src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/WinUI/Properties) to the windows folder and see if it helps?

**Reply by jeromelaban** [reply]
@nickrandolph

**Reply by qizles** [reply]
I'm having the same problem, MSIX package works but self contained does not.

Your solution only contains the MSIX setting in the launchSettings.json , so it dosn't help when you try to follow this example https://platform.uno/docs/articles/features/winapp-sdk-specifics.html for un-packed support.

My properties folder has all the necessary files.

**Reply by nickrandolph** [reply]
Set `<WindowsPackageType>None</WindowsPackageType>` for unpackaged debugging to work
